### PR TITLE
issue 1328 Docu keywords

### PR DESCRIPTION
**Pull request by UliSei**
No coding changed, just docu for keywords
go_to
type_secret
select_options_by
drag_and_drop_by_coordinates

### CODE REVIEW

**Comment by aaltat** [comment]
Fixes #1328

File: Browser/keywords/browser_control.py [hunk 1]
```diff
@@ -48,8 +48,9 @@ def go_to(self, url: str, timeout: Optional[timedelta] = None):
         """Navigates to the given ``url``.
 
         ``url`` <str> URL to be navigated to.
+
```
Comment (aaltat) [inline Browser/keywords/browser_control.py:51]: Most likely here is space, at least flake8 says 
`Browser/keywords/browser_control.py:51:1: W293 blank line contains whitespace`
Could you trim the space away.

**Comment by aaltat** [comment]
There is small space issue, could you fix it and then this ready to go.

File: Browser/keywords/browser_control.py (hunk 1 above)
Comment (UliSei) [reply Browser/keywords/browser_control.py]: Done :-)

**Pull request merged**